rfcs/proposed/range_algorithms/README.md

```diff
@@ -0,0 +1,38 @@
+# Support the second portion of the oneDPL Range APIs
+
+## Introduction
+Based on statistics (observing C++ code within github.com) for the usage of popular algorithms, the following
+range-based APIs are suggested to be implemented next in oneDPL.
+`fill`, `generate`, `move`, `replace`, `replace_if`, `remove`, `remove_if`, `mismatch`, `minmax_element`, `minmax`,
```

Comment:
Let's remove `generate` at this time. There is a concern that the interface may change in p3179 in further revisions.